components/empty-state/empty-state-text.js

```diff
@@ -0,0 +1,88 @@
+import '../colors/colors.js';
+import '../button/button-subtle.js';
+import { css, html, LitElement } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * ADD DESCRIPTION HERE
+ * @slot - You can place your own custom action component in lieu of the actionText property. The component in the slot will only render if actionText is not defined. Only a d2l-button-subtle or d2l-link component will render in this slot.
+ * @fires d2l-empty-state-action - Dispatched when the action button is clicked
+ */
+class EmptyStateText extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * A description giving details about the empty state
+			 * @type {String}
+			 */
+			description: { type: String },
+			/**
+			 * Optional: The action text to be used in the subtle button
+			 * @type {String}
+			 */
+			actionText: { type: String, attribute: 'action-text' }
+		};
+	}
+
+	static get styles() {
+		return css`
+			div {
+				border: 1px solid var(--d2l-color-mica);
+				border-radius: 6px;
+				padding: 1.2rem 1.5rem;
+			}
+			d2l-button-subtle {
+				margin-top: 1px;
+			}
+			:host([dir="rtl"]) span {
+				padding-left: 0.5rem;
+				padding-right: 0;
+			}
+			::slotted(*) {
+				display: none;
+			}
+			::slotted(d2l-button-subtle:first-child),
+			::slotted(d2l-link:first-child) {
+				display: inline;
+			}
+			span {
```

Comment:
`d2l-button-subtle` uses the typography `labelStyles` which has a static font-size of `0.7rem`. In order for the description text to look good in-line next to the button, I have also set the font-size of the description to a static `0.7rem`. The downside to this is that font-size will not change at different breakpoints.

Reply:
Normally when we render the `0.7rem` font-size it is by using [bodySmallStyles](https://github.com/BrightspaceUI/core/blob/main/components/typography/styles.js#L65), but it renders with a tungsten color, so we'd be deviating from our typography here.  I think we should review the issue with using the 0.8rem size with Neroli & Jeff, and call attention to the difference from our typography.